go/vt/vtorc/inst/analysis_dao.go

```diff
@@ -36,7 +36,8 @@ import (
 	"vitess.io/vitess/go/vt/vtorc/util"
 )
 
-var analysisChangeWriteCounter = stats.NewCounter("analysis.change.write", "Number of times analysis has changed")
+// The metric is registered with a deprecated name. The old metric name can be removed in v21.
+var analysisChangeWriteCounter = stats.NewCounterWithDeprecatedName("AnalysisChangeWrite", "analysis.change.write", "Number of times analysis has changed")
```

Comment:
What would we do in v21? We have two options: either revert to using NewCounter with the new name, or replace the old name with empty string while still calling the new function.

Reply:
Also, can/should we delete these new functions in v21?

Reply:
I think we should revert to `NewCounter` and remove the functions. That would be cleaner.